components/concept/sync/src/main/java/mozilla/components/concept/sync/OAuthAccount.kt

```diff
@@ -53,16 +37,25 @@ enum class InFlightMigrationState {
     NONE,
 
     /**
-     * "Copy" in-flight migration present. Can retry migration via [OAuthAccount.retryMigrateFromSessionTokenAsync].
+     * "Copy" in-flight migration present. Can retry migration via [OAuthAccount.retryMigrateFromSessionToken].
      */
     COPY_SESSION_TOKEN,
 
     /**
-     * "Reuse" in-flight migration present. Can retry migration via [OAuthAccount.retryMigrateFromSessionTokenAsync].
+     * "Reuse" in-flight migration present. Can retry migration via [OAuthAccount.retryMigrateFromSessionToken].
      */
     REUSE_SESSION_TOKEN
 }
 
+/**
+ * Data structure describing FxA and Sync credentials necessary to sign-in into an FxA account.
+ */
+data class AuthInfo(
```

Comment:
naming nit: this is a very general name for data that's specific to one very narrow purpose, one could be forgiven for thinking they need an "`AuthInfo`" for all sorts of things. It may be worth giving it a more narrowly-scoped name like `MigratingAccountInfo` or similar.

Reply:
Yup, renamed!